### PR TITLE
Teach sky.Paragraph how to actually compute layout

### PR DESCRIPTION
**Pull request by abarth**
Previously layout was exiting early because the frame view
was null. Now we actually compute some layout and paint the
text. This patch makes paragraph_builder_test pass.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: sky/engine/core/text/Paragraph.cpp [hunk 1]
```diff
@@ -19,46 +21,51 @@ Paragraph::~Paragraph()
...
 
 void Paragraph::paint(Canvas* canvas, const Offset& offset)
 {
+    // Very simplified painting to allow painting an arbitrary (layer-less) subtree.
+    GraphicsContext context(canvas->skCanvas());
+    Vector<RenderBox*> layers;
+    PaintInfo paintInfo(&context, m_renderView->absoluteBoundingBoxRect(), m_renderView.get());
+    LayoutPoint paintOffset(offset.sk_size.width(), offset.sk_size.height());
+    m_renderView->RenderBox::paint(paintInfo, paintOffset, layers);
+    // Note we're ignoring any layers encountered.
```
Comment (Hixie) [inline sky/engine/core/text/Paragraph.cpp]: add a todo or file an issue or something on what needs doing here just in case someone else has to take over

**Comment by Hixie** [comment]
Not sure about the TextDecoration thing being a vector, but LGTM.

**Comment by abarth** [comment]
Yeah, we should probably make it a bitfield.

**Pull request merged**